.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = true
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
+edition = "2018"
```

Comment:
2015 is the default for this one, but I changed it to 2018, because we've already gone to the effort of converting the repo over to be 2018 compliant.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#edition